assume/units/powerplant.py

```diff
@@ -136,7 +161,7 @@ def calculate_operational_window(
 
         if product_type == "energy":
             return self.calculate_energy_operational_window(start, end)
-        elif product_type in {"capacity_pos", "capacity_neg"}:
+        elif product_type in ["capacity_pos", "capacity_neg"]:
```

Comment:
I don't mind, but isn't a set more correct here (unordered, unique values)?
In-Set checks are also faster than lists, but this does not matter at all with less than 10 values (as a rule of thumb)